easybuild/easyconfigs/p/PyMC3/PyMC3-3.8-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,47 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'PyMC3'
+version = '3.8'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://docs.pymc.io/'
+description = """Probabilistic Programming in Python: 
+Bayesian Modeling and Probabilistic Machine Learning with Theano"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),  # for numpy
+    ('Theano', '1.0.4', versionsuffix),
+    ('matplotlib', '3.1.1', versionsuffix),
+    ('h5py', '2.10.0', versionsuffix),
+    ('tqdm', '4.41.1'),
+    ('netcdf4-python', '1.5.3', versionsuffix),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('xarray', '0.15.1', {
+        'checksums': ['64e3138d87b641e22fe7a003c94abc685896b247b63e434505c1e6b38c91a8fb'],
+    }),
+    ('arviz', '0.7.0', {
+        'checksums': ['6872abec18e457c62e2f05591d89befd521b6e680fc57ea6560c6dd12da4f62b'],
+    }),
+    ('patsy', '0.5.1', {
+        'checksums': ['f115cec4201e1465cd58b9866b0b0e7b941caafec129869057405bfe5b5e3991'],
+    }),
+    (name, version, {
+        'source_tmpl': 'pymc3-%(version)s.tar.gz',
```

Comment:
you can avoid this line by using `('pymc3', version, {`

Reply:
eh,had that originally, but I was inspired by other existing blocks to use source_tmpl. Will change that.